helm_helpers_test.go

```diff
@@ -8,11 +8,11 @@ import (
 
 func setupTestCase(t *testing.T) func(t *testing.T) {
 	t.Log("setup test case")
-	os.MkdirAll("/tmp/helmsman-tests/myapp", os.ModePerm)
-	os.MkdirAll("/tmp/helmsman-tests/dir-with space/myapp", os.ModePerm)
+	os.MkdirAll(os.TempDir()+"/helmsman-tests/myapp", os.ModePerm)
+	os.MkdirAll(os.TempDir()+"/helmsman-tests/dir-with space/myapp", os.ModePerm)
 	cmd := command{
-		Cmd:         "bash",
-		Args:        []string{"-c", "helm create  '/tmp/helmsman-tests/dir-with space/myapp'"},
+		Cmd:         "helm",
+		Args:        []string{"create", os.TempDir() + "/helmsman-tests/dir-with space/myapp"},
```

Comment:
Single quote `'` are removed here.

Reply:
That is because the 'qoute' hack is no longer needed.